src/signals/incident-management/containers/IncidentDetail/components/StatusForm/index.test.js

```diff
@@ -197,4 +197,26 @@ describe('signals/incident-management/containers/IncidentDetail/components/Statu
     expect(queryByTestId('statusFormSendEmailField')).not.toBeInTheDocument();
     expect(getByText(DEELMELDING_EXPLANATION)).toBeInTheDocument();
   });
+
+  it('hides the  send checkbox when the excludes state is selected', async () => {
+    const { queryByTestId } = render(renderWithContext());
+
+    const defaultRadioButton = queryByTestId('status-m');
+
+    await EMAIL_CHECKBOX_EXCLUDED_STATES.forEach(state => {
```

Comment:
Why is `await` necessary here? `forEach` Is not an `async` function.

Reply:
Is not needed. I have removed it!